frontend/src/components/review/CommentsPanel.tsx

```diff
@@ -13,19 +13,44 @@ export type CommentPanelProps = {
 
 const CommentsPanel = ({ storyTranslationId }: CommentPanelProps) => {
   const [comments, setComments] = useState<CommentResponse[]>([]);
-  const commentsQuery = buildCommentsQuery(storyTranslationId);
+  const [filterIndex, setFilterIndex] = useState(0);
+
+  const filterOptions = ["Show All", "Show Resolved", "Show Unresolved"];
+  const filterToResolved = [null, true, false];
+  const commentsQuery = buildCommentsQuery(
+    storyTranslationId,
+    filterToResolved[filterIndex],
+  );
   useQuery(commentsQuery.string, {
     fetchPolicy: "cache-and-network",
     onCompleted: (data) => {
       setComments(data.commentsByStoryTranslation);
     },
   });
+
+  const handleSelectChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
I'm a bit partial to more descriptive names - `handleCommentFilterSelectChange`?